packages/test/src/test-integration-update.ts

```diff
@@ -17,6 +25,8 @@ const test = makeTestFunction({
       extraArgs: [
         '--dynamic-config-value',
         `history.longPollExpirationInterval="${LONG_POLL_EXPIRATION_INTERVAL_SECONDS}s"`,
+        '--dynamic-config-value',
```

Comment:
I don't think you need that one, you already added it to defaultDynamicConfigOptions.

Reply:
Removed